Files/UserControls/StatusBarControl.xaml

```diff
@@ -160,9 +160,10 @@
                                     Style="{StaticResource ToggleButtonRevealStyle}"
                                     ToolTipService.ToolTip="Details View">
                                     <FontIcon
-                                        FontFamily="{StaticResource CustomGlyph}"
+                                        Margin="0,2,0,0"
```

Comment:
Instead of setting a margin, can you use the vertical alignment property?

Reply:
For some reason it doesn't work

![Captura de pantalla 2021-03-07 170516](https://user-images.githubusercontent.com/61259627/110253128-7965b600-7f67-11eb-91a1-5f1151c3a74e.png)